components/org.wso2.carbon.identity.scim2.common/src/main/java/org/wso2/carbon/identity/scim2/common/impl/SCIMUserManager.java

```diff
@@ -444,7 +463,7 @@ private boolean canPaginate(int offset, int limit) {
      * @return Paginated usernames list
      * @throws CharonException Pagination not support
      */
-    private String[] listUsernamesAcrossAllDomains(int offset, int limit, String sortBy, String sortOrder)
+    private String[] listUsernamesAcrossAllDomains(int offset, Integer limit, String sortBy, String sortOrder)
```

Comment:
Is it necessary to change this one

Reply:
fixed:8f8c20292c4fac76778a8aa95490675023d7d063